diagnostics-app/src/app/telemetry/telemetry.module.ts

```diff
@@ -25,4 +25,4 @@ import { CpuComponent } from './cards/cpu/cpu.component';
   imports: [CommonModule, SharedModule],
   schemas: [CUSTOM_ELEMENTS_SCHEMA],
 })
-export class DashboardModule {}
+export class TelemetryModule {}
```

Comment:
Please update line 6.

Reply:
Done. I also updated the fileoverview parts in "cpu.component.ts", "memory.component.ts", and "storage.component.ts".